storybook/stories/ServerItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { ScrollView } from 'react-native';
+
+import { themes } from '../../app/constants/colors';
+import ServerItemComponent from '../../app/presentation/ServerItem';
+import StoriesSeparator from './StoriesSeparator';
+
+let _theme = 'light';
+
+const item = {
+	name: 'https://open.rocket.chat/',
+	id: 'gerzon.canario',
+	iconURL: 'https://open.rocket.chat/images/logo/android-chrome-512x512.png'
+};
+
+
+const ServerItem = props => (
+	<ServerItemComponent
+		theme={_theme}
+		{...props}
+	/>
+);
+
+// eslint-disable-next-line react/prop-types
+const Separator = ({ title }) => <StoriesSeparator title={title} theme={_theme} />;
+
+// eslint-disable-next-line react/prop-types
+export default ({ theme }) => {
```

Comment:
We use Storybook in 2 ways currently:
- the way we used to do, which is this one (migration is WIP)
- the new way (like `List` and `HeaderButton`), which separates stories completely creating more detailed tests

Can you change your stories to this new way?

Also, a good practice is to add stories trying to break your component.
For example, add a long test to the title prop trying to break line.

Don't forget to add stories for different themes.